File: 06.PHP/pg-226-230/pg_226_02.php
<?php
echo("");
echo("PHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHP<br>");
echo("READ AN INTEGER AND DISPLAY ALL EVEN NUMBERS BETWEEN ONE AND THE GIVEN NUMBER<br>");
echo("PHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHP<br><br>");
echo("");


echo("Write an integer: ");
$num = 33;
echo($num . "<br>");

if($num < 0)
    {
        $num *= -1;
    }
$i = 1;
/*
for($i=1; $i<=$num; $i++)
    {
        if($i % 2 == 0)
        {
            echo($i . ", ");
        }
    }
*/

while($i <= $num)
    {
        if($i % 2 == 0)
        {
            echo($i . "-");
        }
        $i ++;
    }
?>

File: 02.PYTHON/pg-226-230/pg_226_02.py
print("");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYP");
print("READ AN INTEGER AND DISPLAY ALL EVEN NUMBERS BETWEEN ONE AND THE GIVEN NUMBER");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYP");
print("");

num = int(input("Write an integer: "));
i = 1;

if(num < 0):
    num *= -1;
"""
for i in range(1, num, 1):
    if(i % 2 == 0):
        print(i, end =", ");
"""

while(i<=num):
    if(i % 2 == 0):
        print(i, end="- ")
    i+=1;
